types/blob/blob.go

```diff
@@ -0,0 +1,40 @@
+package blob
+
+import (
+	"encoding/json"
+
+	"github.com/celestiaorg/nmt/namespace"
+	rpcproto "github.com/rollkit/celestia-openrpc/proto/celestia/openrpc"
```

Comment:
why not just have `celestia-openrpc/proto/blob`? i mean blob.proto?

Reply:
Works 👍🏽